NuGet.config

```diff
@@ -4,9 +4,10 @@
   <packageSources>
     <clear />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
-    <add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
     <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
```

Comment:
Just realized the first failure was due to Microsoft.CodeAnalysis.NetAnalyzers not being found. It lives in dotnet8, so I think we can remove dotnet8-transport.

```suggestion
```